source/install/install-tar.rst

```diff
@@ -1,8 +1,260 @@
+      <ul>
+        <li>Hardware: 1 vCPU/core with 2GB RAM (support for up to 1,000 users)</li>
+        <li>Database: PostgreSQL v11+</li>
+        <li>Network:
+          <ul>
+            <li>Application 80/443, TLS, TCP Inbound</li>
+            <li>Administrator Console 8065, TLS, TCP Inbound</li>
+            <li>SMTP port 10025, TCP/UDP Outbound</li>
+          </ul>
+        </li>
+      </ul>
+    </div>
+
+  </div>
+
+You can install the Mattermost Server on any 64-bit Linux system using the tarball. Deployment includes 4 steps: `download <#download-the-latest-mattermost-server-tarball>`__, `install <#install>`__, `setup <#setup>`__, and `update <#updates>`__.
+
+.. important::
+
+    - This is the most flexible installation method, but it comes with the highest effort, normally favored by advanced system adminsitrators. 
+    - If you are running the Mattermost Server and database a single system, we recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ as this greatly reduces setup and ongoing maintenance, and using the Mattermost PPA for a scalable installation. More modern installation methods such as the Mattermost Helm Chart or Kubernetes Operator are available and are highly recommended.
+
+
+Download the latest Mattermost Server tarball
+---------------------------------------------
+
+In a terminal window, ssh onto the system that will host the Mattermost Server. 
+
+Using ``wget``, download the Mattermost Server release you want to install.
+
+Download the latest release
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+  <div class="mm-code-copy mm-code-copy--long" data-click-method="Tarball" data-click-command="Download the latest release">
+
+    <div class="mm-code-copy__wrapper">
+      <code class="mm-code-copy__text mm-code-copy__trigger" data-click-el="Snippet">
+        wget https://releases.mattermost.com/7.10.0/mattermost-7.10.0-linux-amd64.tar.gz
+      </code>
+      <span class="mm-code-copy__copied-notice">Copied to clipboard</span>
+    </div>
+
+    <button class="mm-button mm-code-copy__trigger" data-click-el="Button">
+      <svg aria-hidden="true" width="17" height="18" viewBox="0 0 17 18" fill="none" xmlns="http://www.w3.org/2000/svg"><rect x="0.5" y="0.5" width="10.2972" height="10.8284" rx="0.5" stroke="white"/><rect x="6.1489" y="6.41418" width="10.2972" height="10.8284" rx="0.5" stroke="white"/></svg>
+      <span>Copy to clipboard<span>
+    </button>
+
+  </div>
+
+Download the current Extended Support Release (ESR)
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. raw:: html
+
+  <div class="mm-code-copy mm-code-copy--long" data-click-method="Tarball" data-click-command="Download the current ESR">
+
+    <div class="mm-code-copy__wrapper">
+      <code class="mm-code-copy__text mm-code-copy__trigger" data-click-el="Snippet">
+        wget https://releases.mattermost.com/7.8.3/mattermost-7.8.3-linux-amd64.tar.gz
```

Comment:
@amyblais would it be possible to add to your release process a step to amend this URL please? We have chosen to provide the latest and link it to the copy-action button for ease for admins (we used to ask them to create their own link from the archive page).

Reply:
Added to the release checklist 👍